<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>571</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="lbId">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cbEstado">
      <item>
       <property name="text">
        <string>FUNCIONANDO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ROTA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EN REPARACION </string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbProveedor">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Proveedor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbEstado">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Estado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leModel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cbProveedor"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbModel">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Model</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Productividad (Unidad/min) </string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Producto</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbProducto"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tiempo (min)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sbTiempo"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leNombre">
      <property name="inputMethodHints">
       <set>Qt::ImhPreferUppercase|Qt::ImhUppercaseOnly</set>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="leObservaciones">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="lbObservaciones">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>104</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Observaciones</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>512</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btExit">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btSave">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btExit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
